src/utils/contacts.js

```diff
@@ -0,0 +1,34 @@
+import { Map } from 'immutable';
```

Comment:
these contacts should better be placed in the src/constants/app file.
utils is used for utility functions. please, move.

Reply:
moved